apify-docs-theme/src/theme/custom.css

```diff
@@ -74,6 +159,20 @@ html[data-theme='dark'] {
 
     --ifm-heading-color: #242736;
 
+    --ifm-leading: calc(var(--ifm-leading-desktop) * 1.6rem);
+    --ifm-list-margin: 1.6rem;
+    --ifm-pre-padding: 1.6rem;
+
+    --ifm-hr-margin-vertical: 2.4rem;
+
+    /* Comes from the Apify design theme content typography tokens */
```

Comment:
The font sizes are correct but when we were defining them we forgot that H1 basically doesn't exist in markdown and you use H2 as the "main" heading. So we should push down the font-sizes so that H1 size is not H2 and come up with a bigger H1 size.